doc/source/getting_started/comparison/includes/copies.rst

```diff
@@ -0,0 +1,20 @@
+Most pandas operations return copies of the ``Series``/``DataFrame``. To make the changes "stick",
+you'll need to either:
+
+* Assign back to the same variable
+
+   .. code-block:: python
+
+      df = df.sort_values("col1")
+
+* Assign to a new variable
+
+   .. code-block:: python
+
+      sorted_df = df.sort_values("col1")
+
+* Use the ``inplace=True`` keyword argument, where available
```

Comment:
frown on this 

in place is never recommended

Reply:
A few options:

- Say "in order of preference"
- Mention `inplace` as "you will come across this, but don't use it"
- Remove mention entirely

Strong feelings?

Reply:
i ok with mentioning it, but put it in a note and point to this link: https://pandas.pydata.org/pandas-docs/dev/user_guide/indexing.html#indexing-view-versus-copy

Reply:
Done, though the "Returning a view versus a copy" section doesn't actually mention `inplace`. I'd submit a pull request to add an explanation, but frankly, that's one area of pandas that's still pretty fuzzy to me.

Reply:
sure that would be great

Reply:
What I mean is "I'm not sure I understand it fully myself, so would love an explanation from someone else" 😊

Reply:
oh i c. check this blog by @TomAugspurger https://tomaugspurger.github.io/method-chaining.html

generaly `inplace` doesn't do a whole lot, is not idiomatic, and cause  copy/view issues.